src/test/java/com/teragrep/pth10/statsTransformationTest.java

```diff
@@ -190,7 +190,6 @@ void statsTransform_AggRate_Test() {
     void statsTransform_AggEarliest_Test() {
```

Comment:
Can we have a test for the `| stats earliest(_raw), latest(_raw)` combo?

Reply:
added